.claude/database-ops/SCHEMA.md

```
@@ -0,0 +1,391 @@
+
+-- Hourly ingest rate
+SELECT toStartOfHour(ingested_at) as hour, count() as events
+FROM eventsWHERE ingested_at >= now() - INTERVAL 24 HOUR
+GROUP BY hour ORDER BY hour;
+```
+
+---
+
+## Environment Variables (`.env`)
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)